packages/grafana-ui/src/components/Spinner/Spinner.tsx

```diff
@@ -80,7 +82,17 @@ export const Spinner = ({
         className
       )}
     >
-      <Icon className={cx('fa-spin', iconClassName)} name="spinner" size={size} aria-label="loading spinner" />
+      <Icon
+        className={cx(
+          {
+            'fa-spin': !prefersReducedMotion,
+          },
+          iconClassName
+        )}
+        name={iconName}
+        size={size}
+        aria-label="loading spinner"
```

Comment:
While where here should we get rid of this aria-label? 🥺

Reply:
does it need replacing with anything or are we just straight removing it? 🤔

Reply:
Hmm... after a brief look, about 75% use the spinner without a text label (and so would require a aria-label). Maybe lets just change it to 'loading' for now?